lib/UserManagement.php

```diff
@@ -164,12 +164,13 @@ public function deleteUser($userId)
      *
      * @return \WorkOS\Resource\Response
      */
-    public function addUserToOrganization($userId, $organizationId)
+    public function createOrganizationMembership($userId, $organizationId)
     {
-        $userOrganizationPath = "users/{$userId}/organizations";
+        $userOrganizationPath = "user_management/organization_memberships";
```

Comment:
nit: i would rename this, maybe `organizationMembershipsPath`? (applies to the other methods here too)

Reply:
Called it $path, simplier